gh-106554: Replace `_BaseSelectorImpl._key_from_fd` with a simple `get` (#106555)
python/cpython: Misc/NEWS.d/next/Library/2023-07-09-01-59-24.gh-issue-106554.37c53J.rst

> Reduce selector overhead

Author: ```suggestion
Reduce selector overhead by switching `SelectSelector` to use a `dict` `.get()` for `fd`s
```

> Reduce selector overhead by switching `SelectSelector` to use a `dict` `.get()` for `fd`s

Author: ```suggestion
Reduce selector overhead by switching ``SelectSelector`` to use a ``dict`` ``.get()`` for ``fd``s
```

> Reduce selector overhead by switching ``SelectSelector`` to use a ``dict`` ``.get()`` to fetch file descriptors.

Author: ```suggestion
Reduce selector overhead by switching ``SelectSelector`` to use a ``dict`` ``.get()`` to lookup file descriptors.
```

> Reduce selector overhead by switching ``SelectSelector`` to use a ``dict`` ``.get()`` to lookup file descriptors.

Reviewer: ```suggestion
:mod:`selectors`: Reduce Selector overhead by using a ``dict.get()`` to lookup file descriptors.
```

Author: Thanks
